src/components/Contacts/ContactDetail/ContactDetailTab/ContactDetailTab.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { gqlMock } from '../../../../../__tests__/util/graphqlMocking';
+import {
+  ContactDetailDocument,
+  ContactDetailQuery,
+} from '../ContactDetail.generated';
+import { ContactDetailTab } from './ContactDetailTab';
+
+export default {
+  title: 'Contacts/Tab/ContactDetailTab',
+  component: ContactDetailTab,
+};
+
+export const Default: Story = () => {
+  const contact = gqlMock<ContactDetailQuery>(ContactDetailDocument, {
+    mocks: {},
+  });
```

Comment:
```suggestion
  const contact = gqlMock<ContactDetailQuery, ContactDetailQueryVariables>(
    ContactDetailDocument,
    {
      variables: { accountListId: 'account-list-1', contactId: 'contact-1' },
    },
  );
```

Reply:
Thanks, @OzzieOrca that is exactly what I was missing.